app/src/main/java/com/example/habitsmasher/ui/history/EditHabitEventDialog.java

```diff
@@ -92,11 +112,35 @@ public void onClick(View view) {
                         eventComment,
                         _editHabitEvent.getId());
                 _errorText.setText("");
-                _habitEventListFragment.updateListAfterEdit(editedHabitEvent,_index);
+                _habitEventListFragment.editHabitEvent(editedHabitEvent,_index, _selectedImage);
 
                 // Close dialog
                 getDialog().dismiss();
             }
         });
     }
+
+    private void fetchEventImageFromDB() {
+        FirebaseStorage storage = FirebaseStorage.getInstance();
+        StorageReference storageReference = storage.getReference();
+
+        StorageReference ref = storageReference.child("images/" + _userId + "/" + _parentHabit.getId() + "/" + _editHabitEvent.getId() + "/eventImage");
```

Comment:
We seem to be building up the "path" a lot manually. It would be nice to have a simple helper method that returns the path string so that this is readable and other people know exactly what's going on here.